### ISSUE TITLE
closing port command not working on windows

### ISSUE THREAD

**Issue by BananaAcid** [Wscats/compile-hero#10]
Refering to [vscode-extension/src/extension.ts#L226](https://github.com/Wscats/compile-hero/blob/6d54b9a0b00cfd6c38b9ce6b4152fe0150c8dbbc/vscode-extension/src/extension.ts#L226) -> this does not work on windows.

Something like [[this]](https://developers.de/blogs/indraneel/archive/2017/10/18/kill-a-process-in-windows-by-port-number.aspx) could work.

**Comment by Wscats** [comment]
Yes, there is no window support yet @BananaAcid

**Comment by BananaAcid** [comment]
Yes, please do an OS check before trying to `execute` to not trigger an error kill the plugin host as well.
